extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/Endpoints.java

```diff
@@ -212,14 +211,37 @@ public void handle(Throwable t) {
                     public void handle(Void event) {
                         endpoint.doOnError(t).subscribe().with(
                                 v -> LOG.debugf("Error [%s] processed: %s", t.getClass(), connection),
-                                t -> LOG.errorf(t, "Unhandled error occured: %s", t.toString(),
+                                t -> LOG.errorf(t, "Unhandled error occurred: %s", t.toString(),
                                         connection));
                     }
                 });
             }
         });
     }
 
+    private static void logFailure(Throwable throwable, String message, WebSocketConnectionBase connection) {
+        if (isWebSocketIsClosedFailure(throwable)) {
+            LOG.debugf(throwable,
+                    message + ": %s",
+                    connection);
+        } else {
+            LOG.errorf(throwable,
+                    message + ": %s",
+                    connection);
+        }
+    }
+
+    private static boolean isWebSocketIsClosedFailure(Throwable throwable) {
+        if (throwable == null) {
```

Comment:
Could we add a check that `WebSocketConnectionBase#isClosed()` returns `true`?

I wonder if we should create a Vertx issue to introduce `ConnectionClosedException` or something like that...

Reply:
> Could we add a check that WebSocketConnectionBase#isClosed() returns true?

Done

> I wonder if we should create a Vertx issue to introduce ConnectionClosedException or something like that.

@vietj would you be willing to have that in Vertx?